src/components/AddItem.js

```diff
@@ -1,7 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { writeToFirestore } from '../lib/firebase';
 
 const AddItem = () => {
-  return <h1>THIS IS THE ADD ITEM</h1>;
+  let [name, setName] = useState('');
+  let [frequency, setFrequency] = useState('');
+
+  function handleSubmitForm(event) {
+    event.preventDefault();
+    if (name.length > 0) {
+      writeToFirestore('put-token-here', {
+        name,
+        frequency,
+        lastPurchaseDate: null,
+      });
+      alert(`${name} has been successfully added to your list.`);
+      setName('');
+    }
+  }
+
+  return (
+    <form onSubmit={event => handleSubmitForm(event)}>
+      <label htmlFor="add-shopping-list-item">Item Name</label>
+      <input
+        value={name}
+        onChange={e => setName(e.target.value)}
+        id="add-shopping-list-item"
+      />
+      <fieldset>
+        <legend>How soon will you buy this again?</legend>
+        <div>
+          <input
+            onChange={e => setFrequency(e.target.value)}
+            type="radio"
+            id="soon"
+            name="frequency"
+            value={7}
+          />
+          <label htmlFor="soon">Soon</label>
+          <input
+            onChange={e => setFrequency(e.target.value)}
+            type="radio"
+            id="kind-of-soon"
+            name="frequency"
+            value={14}
+          />
+          <label htmlFor="kind-of-soon">Kind of Soon</label>
+          <input
+            onChange={e => setFrequency(e.target.value)}
+            type="radio"
+            id="not-soon"
+            name="frequency"
+            value={30}
+          />
+          <label htmlFor="not-soon">Not Soon</label>
+        </div>
+      </fieldset>
```

Comment:
This looks great! Hadn't seen the fieldset tag before this past week. Makes perfect sense to use it with a group of possible choice fields.

Reply:
Great use of semantic HTML! 👍    Here's a link to `<fieldset>` in the docs for those wishing to dive in a little deeper: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset